.github/ISSUE_TEMPLATE/new_scraper.md

```diff
@@ -7,14 +7,7 @@ assignees: ''
 
 ---
 
-We always enjoy adding support for new websites to the library!
```

Comment:
(I mean, it is a nice feeling when tests pass - I sorta feel this line has a bit of a Blade Runner off-world colonies marketing voice to it, though)